test/v3/createAmendAnnualSubmission/def1/Def1_CreateAmendAnnualSubmissionValidatorSpec.scala

```diff
@@ -19,13 +19,12 @@ package v3.createAmendAnnualSubmission.def1
 import api.models.domain.ex.MtdNicExemption
 import api.models.errors.{Class4ExemptionReasonFormatError, RuleBothAllowancesSuppliedError, RuleBuildingNameNumberError}
 import config.MockSeBusinessFeatureSwitches
-import play.api.Configuration
 import play.api.libs.json.{JsNumber, JsValue, Json}
-import shared.utils.UnitSpec
 import shared.config.MockSharedAppConfig
 import shared.models.domain.{BusinessId, Nino, TaxYear}
 import shared.models.errors._
 import shared.models.utils.JsonErrorValidators
+import shared.utils.UnitSpec
 import v3.createAmendAnnualSubmission.CreateAmendAnnualSubmissionValidatorFactory
 import v3.createAmendAnnualSubmission.def1.model.request._
 import v3.createAmendAnnualSubmission.model.request.{CreateAmendAnnualSubmissionRequestData, Def1_CreateAmendAnnualSubmissionRequestData}
```

Comment:
Please remove `with MockSharedAppConfig with MockSeBusinessFeatureSwitches` as it is not required, thanks.